### PR TITLE
Edited collation tags and added a function to return objects given a string industry

### PR DESCRIPTION
**Pull request by liangdanica**
- collation tags (#44)
-  function to return objects given a string industry (#2)
-- I'm currently working on making it so that the parameter is an array in order to search for users of more than one industry

### CODE REVIEW

File: src/search.sjs [hunk 1]
```diff
@@ -64,20 +64,42 @@ if (input.features === false) {
...
+  features = jsearch.facets([jsearch.facet('Features', 'features').slice(0,215)]).result();
 }
 
-if (input.features === true) {
-  features = jsearch.facets([jsearch.facet('Features', 'features').slice(0,215)]).result();
+if (input.industries !== '') {
+  // function to find all users in a given industry
+  // 'ind' as a string parameter represents the industry
+  // outputs array of GeoJSON objects
+  xdmp.log("finding");
```
Comment (dmcassel) [inline src/search.sjs]: Some log statements are worth leaving in the code, but if you're going to, it should have more information. This looks like a debugging log statement, so it should probably be removed.

File: src/search.sjs [hunk 2]
```diff
@@ -64,20 +64,42 @@ if (input.features === false) {
 
 var industries = {};
 var features = {};
+var found = [];
 
-if (input.industries === true) {
+if (input.firstLoad === true) {
```
Comment (dmcassel) [inline src/search.sjs]: wouldn't you want to get the facets for each query, not just the first load?

**Comment by liangdanica** [comment]
Industries and features also now ordered by frequency. Added an orderBy method to the facet definitions.

File: src/search.sjs [hunk 3]
```diff
@@ -61,14 +61,21 @@ if (input.features === false) {
 
   searchResults = cts.search(geoQueryJson).toArray();
 }
-
+xdmp.log(searchResults);
```
Comment (dmcassel) [inline src/search.sjs]: This looks like one to remove

**Comment by liangdanica** [comment]
Made the count visible, so now features and industries are shown like this....
Cubix _(2)_

**Pull request merged**